dm/loader/lightning.go

```diff
@@ -118,7 +120,39 @@ func (l *LightningLoader) Init(ctx context.Context) (err error) {
 		err = nil
 	})
 	l.checkPoint = checkpoint
-	l.toDB, l.toDBConns, err = createConns(tctx, l.cfg, 1)
+	toCfg, err := l.cfg.Clone()
+	if err != nil {
+		return err
+	}
+	if toCfg.To.Session == nil {
+		toCfg.To.Session = make(map[string]string)
+	}
+	toCfg.To.Session["time_zone"] = "+00:00"
```

Comment:
seem @glorv  already adapt time_zone in https://github.com/pingcap/dm/pull/2217, maybe add some comments here for later conficts

Reply:
We can keep this change and I will handle this when migrating dm#2217 to this repo.

Reply:
Done